### DISCUSSION TITLE
Exporting Full Documentation to single file

### DISCUSSION THREAD

**Discussion by Hokyjack** [quasarframework/quasar#16828] [General - Components / Directives / etc]
Hello I would like to train a supporting chatbot for Quasar Framework. I would need to collect the whole documentation and use it as a data. Is it possible to build the documentation easily (and how) preferably into a single document (e.g. markdown or pdf)? Thank you.

**Comment by dongwa** [comment]
Perhaps the quasar documentation source code contains the markdown file you want

https://github.com/quasarframework/quasar/tree/dev/docs/src/pages
